docs/source/guides/writer/chapters/writing.rst

```diff
@@ -1107,6 +1103,7 @@ status and will not make the Job to exit with a non-0 status. Example::
 In a system missing the `iperf` package but with `gcc` installed in
 the correct version, the result will be::
 
+    $ avocado run  cancel_test.py
```

Comment:
Nitpick: there's a double space between `run` and `cancel_test.py`.

Reply:
good catch, thanks